client/src/modules/report-group/index.html

```diff
@@ -0,0 +1,170 @@
+<div class="flex-header static">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.ADMIN" | translate }}</li>
+      <li class="static">{{ "Reporting Group" | translate }}</li>
+    </ol>
+
+    <div class="toolbar">
+      <div class="toolbar-item">
+        <a  class="btn btn-default pull-right"
+          ui-sref="emailreport"
+          ui-sref-opts="{notify:false}">
+            <i class="fa fa-plus"></i>
+            <span  translate=""  class="ng-scope">
+              Email Report
+            </span>
+        </a>
+      </div>
+
+    </div>
+  </div>
+</div>
+
+
+
+<div class="flex-content">
+  <div class="container-fluid" id="anchor">
+
+  <div class="col-lg-12">
+    <!-- start new Reporting group form -->
+    <div class="col-lg-5">
+    <h4> {{"New reporting group"}}</h4>
```

Comment:
There is no difference between the following two lines in output:
```html
<h4>{{ "New reporting Group" }}</h4>
<!-- versus -->
<h4>New reporting Group</h4>
```

When you write the `{{ }}` angular syntax, the browser attempts to evaluate what is between the brackets.  Since it's a string, it will simply return the string.

However, any time anything changes on the page, angular will a "digest" cycle.  It will re-evaluate everything in curly braces (`{{ }}`) on the page.  The more curly braces you have, the slower the performance will be.

Since you do not need the braces here, you should remove them.